.github/workflows/python.yml

```diff
@@ -17,7 +17,7 @@ name: Python Agent CI
 on:
   pull_request:
   push:
-    # Run on push to main or PR. 
```

Comment:
It looks like this is just whitespace, so I'll see if I can remove this file from the PR to avoid confusion.